src/plays/news-feed-application/countriesAndCategories.js

```diff
@@ -0,0 +1,330 @@
+export const categories = ["business", "health", "entertainment", "science", "sports", "technology", "general"];
+
+export const countries = [
+    {
+      code: "AR",
```

Comment:
is code3 and number property getting used anywhere if not please remove extra data